app/templates/views/service-settings.html

```diff
@@ -434,7 +434,9 @@ <h2 class="heading-medium">Platform admin settings</h2>
               {% if not current_service.broadcast_channel %}
               Off
               {% else %}
-                {% if current_service.live %}Live{% else %}Training mode{% endif %} - {% if current_service.allowed_broadcast_provider %}{{ current_service.allowed_broadcast_provider|format_mobile_network }} network{% else %}All networks{% endif %} - {% if current_service.broadcast_channel == "test" %}Test channel only{% else %}Public channel{% endif %}
+                {% if current_service.live and current_service.broadcast_channel == "severe" %}Live - Public channel{% endif %}
+                {% if current_service.live and current_service.broadcast_channel == "test" %}Test channel {% if current_service.allowed_broadcast_provider %}({{ current_service.allowed_broadcast_provider|format_mobile_network }} network){% else %} (All networks){% endif %}{%endif%}
```

Comment:
And this should be _Test_, without the word _Network_ (it’s implicit that EE is a network)
```suggestion
                {% if current_service.live and current_service.broadcast_channel == "test" %}Test{% if current_service.allowed_broadcast_provider %}({{ current_service.allowed_broadcast_provider|format_mobile_network }}){% else %} (All networks){% endif %}{%endif%}
```